loops-and-arrays.js

```diff
@@ -26,6 +30,13 @@ const numbers = [2, 4, 5, 29, 38];
 // console.log(numbers) geeft: [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 6, 12, 15, 87, 114 ];
+console.log(numbers);
+numbers[0] = 6;
+numbers[1] = 12;
+numbers[2] = 15;
+numbers[3] = 87;
+numbers[4] = 114;
+console.log(numbers);
 
```

Comment:
Hier hetzelfde. Het is handiger om gebruik te maken van een for loop:
for (let i = 0; i < numbers.length; i++) {
  numbers[i] = numbers[i] * 3;